app/controllers/public_pages_controller.rb

```diff
@@ -17,7 +17,7 @@ def home
     if params[:source].present?
       source_parameter = SourceParameter.find_by(code: params[:source]&.downcase)
       if source_parameter.present?
-        if source_parameter.active?
+        if source_parameter.active? && source_parameter.vita_partner.present?
```

Comment:
I meant to indicate I this check should go one layer up since the `else` statement also uses `source_parameter.vita_partner.name`. I think that's appropriate, because if there's no partner then it means we will just redirect

Reply:
Fixed in second PR